keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/ApplicationLoadBalancerHandlerTests.kt

```diff
@@ -126,7 +126,7 @@ internal class ApplicationLoadBalancerHandlerTests : JUnit5Minutests {
         certificates = null,
         rules = emptyList(),
         defaultActions = listOf(
-          Action(
+          ApplicationLoadBalancerModel.Action(
```

Comment:
nit: I thought generally we tried to go the other way, i.e. import the inner class like it was before?

Reply:
I think IntelliJ screws those up when you move classes

Reply:
In this case it's because we're already importing a class with the same name